src/components/spinner/Spinner.tsx

```diff
@@ -68,13 +71,26 @@ const AminoSpinner = styled.span<SpinnerProps>`
       border-right-color: rgba(255, 255, 255, 0.3);
     }
   }
+
+  &.info {
+    border-top-color: ${theme.blueBase};
+  }
+  &.success {
+    border-top-color: ${theme.greenBase};
+  }
+  &.danger {
+    border-top-color: ${theme.redBase};
+  }
+  &.warning {
+    border-top-color: ${theme.orangeBase};
+  }
 `;
 
-type Color = 'black' | 'white';
+export type SpinnerColor = 'black' | 'white' | ThemeColor;
```

Comment:
What do you think about picking type from the list of color in `src/types/Intent.ts` instead of creating a new type?